CHANGELOG.md

```diff
@@ -31,65 +31,63 @@ These functions and structs can be referenced in both code and blueprints it may
+- You can now select an actor for SpatialOS debugging in-game. Use F9 (by default) to open the Spatial Debugger in-game config menu and then press the `Start Select Actor(s)` button. Hover over an actor with the mouse to highlight and right-click  (by default) to select. You can select multiple actors. To deselect an actor right-click on it a second time. If there are multiple actors under the cursor use the mouse wheel (by default) to highlight the desired actor then right-click to confirm your selection.
 - SpatialWorldSettings is now the default world settings in supported engine versions.
 - Worker SDK version compatibility is checked at compile time. 
-- Unreal GDK now uses SpatialOS 15.0.0.
 - SpatialWorkerFlags has reworked how to add callbacks for flag updates:
   1. `BindToOnWorkerFlagsUpdated` is changed to `RegisterAnyFlagUpdatedCallback` to better differentiate it from the newly added functions for register callbacks. 
   2. `RegisterFlagUpdatedCallback` is added to register callbacks for individual flag updates
   3. `RegisterAndInvokeAnyFlagUpdatedCallback` & `RegisterAndInvokeFlagUpdatedCallback` variants are added that will invoke the callback if the flag was previously set.
 - Overhauled local deployment workflows. Significantly faster local deployment start times (0.1s~ vs 7s~).
   - Switched to using binary (standalone) versions of the Runtime and Inspector which allow us to start deployments much faster.
   - Runtime and Inspector binary fetching is now handled by the GDK. To fetch these binaries you must start the game project at least once.
-  - The Runtime and Inspector will be restarted between each editor session.
+  - The Runtime and Inspector will be restarted between each Editor session.
   - Support for standalone launch configurations introduced, these are now generated by default for your local deployments, worker configurations for local deployments are now stored inside the launch config. 
     - Generated local launch configs can still be found in `<GameProject>\Game\Intermediate\Improbable\<MapName>_LocalLaunchConfig.json`.
   - Cloud deployments still use the older launch config and worker config format, we still generate these automatically for your maps.
     - Generated cloud launch configs can  be found in `<GameProject>\Game\Intermediate\Improbable\<MapName>_CloudLaunchConfig.json`.
   - The Launch Configuration Generator tool can be used to generate either local or cloud launch configs via a checkbox in the GUI.
-  - Spot and SpatialD dependencies have been removed.
```

Comment:
How come this line has been removed?

Reply:
It's a duplicate. Line 30 also includes this:
- Spot and SpatialD (Spatial Service) dependencies have been removed.